src/odin/maneuversbuilder.cc

```diff
@@ -1648,6 +1741,31 @@ void ManeuversBuilder::SetManeuverType(Maneuver& maneuver, bool none_type_allowe
     maneuver.set_type(DirectionsLeg_Maneuver_Type_kFerryExit);
     LOG_TRACE("ManeuverType=FERRY_EXIT");
   }
+  // Process elevator
+  else if (maneuver.elevator()) {
+    maneuver.set_type(DirectionsLeg_Maneuver_Type_kElevatorEnter);
+    LOG_TRACE("ManeuverType=ELEVATOR");
+  }
+  // Process steps
+  else if (maneuver.steps()) {
+    maneuver.set_type(DirectionsLeg_Maneuver_Type_kStepsEnter);
+    LOG_TRACE("ManeuverType=STEPS");
+  }
```

Comment:
i feel like we maybe dont want to call out using the steps in all situations. im a novice compared to @dgearhart but my gut intuition is that there are lots of cases, especially outside on like hiking paths where we dont need to say take the stairs/steps as there is nothign else to do. indoors steps are a disambiguating feature so i could see always calling them out in that case but then my next thought is what about outdoors but like in a dense city around buildings and stuff. how can we tell when it makes sense to call them out and when not? @dgearhart opinions?

Reply:
@kevinkreiser I will have to take a look

Reply:
This makes sense to me. An easy alternative is to call out indoor stairs only, while leaving outdoor steps ([example](https://wiki.openstreetmap.org/wiki/File:Stairs_with_ramp.jpg)) as it is today. It would look like this:
```
  // Steps
  if (prev_edge->IsStepsUse() && prev_edge->indoor()) {
    maneuver.set_steps(true);
  }
```
Still, I am curious on what @dgearhart thinks.

Reply:
yep that is what my gut was saying for a simple approach. maybe we could start with that and evolve to be smarter in a subsequent pr, just so we arent blocked on design there, im not sure on @dgearhart 's availalbility at the moment to spend some brain cells on it.